Harvest.Core/Services/SlothService.cs

```diff
@@ -81,115 +81,29 @@ public async Task<Result<SlothResponseModel>> MoveMoney(int invoiceId)
                 return Result.Error("No expenses found for invoice: {invoiceId}", invoiceId);
             }
 
-            if (!invoice.Expenses.All(e => e.Total > 0))
+            var grandTotal = Math.Round(invoice.Expenses.Select(a => a.Total).Sum(),2);
+            if (grandTotal <= 0)
             {
+                Result.ResultError xx1x = Result.Error("Expenses found with a Total of 0 or less for invoice: {invoiceId}", invoiceId);
                 return Result.Error("Expenses found with a Total of 0 or less for invoice: {invoiceId}", invoiceId);
```

Comment:
This is weird. Why the extra call and variable?

Reply:
Oops. Was trying to figure out a good way to do errors. I'll remove